.github/workflows/e2e_libvirt.yaml

```diff
@@ -79,6 +82,7 @@ jobs:
       - name: Config Libvirt
         run: |
           ./libvirt/config_libvirt.sh
+          echo "CAA_IMAGE=\"${{ inputs.caa_image }}\"" >> libvirt.properties
```

Comment:
As `inputs.caa_image` is non-required input then it might end up writing `CAA_IMAGE=` (empty). I don't think we will be testing a case where CAA_IMAGE is empty, will we? If not, I suggest to make `inputs.caa_image` required.

Reply:
Yep, that makes sense.